test/lit.cfg.py

```diff
@@ -42,10 +42,8 @@
 # excludes: A list of directories to exclude from the testsuite. The 'Inputs'
 # subdirectories contain auxiliary inputs for various tests in their parent
 # directories.
-config.excludes = ['Inputs', 'Examples', 'CMakeLists.txt', 'README.txt', 'LICENSE.txt']
```

Comment:
Why this line is removed?

Reply:
AFAICS, this line is unused because the the `line.suffixes` above states `mlir` only, and there are no MLIR files on those directories. All tests pass without this line, and even the `stdx-ops.mlir` below work, so I'm not sure how much effect that was having.